index.js

```diff
@@ -110,7 +110,7 @@ module.exports = (text, opts) => {
 
 	const contentWidth = widestLine(text) + padding.left + padding.right;
 	const paddingLeft = PAD.repeat(padding.left);
-	const columns = process.stdout.columns;
+	const columns = process.env.STDOUT_COLUMNS || process.stdout.columns;
```

Comment:
I thought about that too, but then the case of running within a child process came along, where you might still have a window wider than 80, in which case neither `left` nor `center` would work. In any case, I am merely helping out here, so whatever you consider to be the right approach is fine.

Let me know and I will update the code.